workflow/scripts/functional_enrichment.py

```diff
@@ -114,7 +115,8 @@ def get_options():
 
     r['qvalue'] = sm.stats.multipletests(r['pvalue'], alpha=0.05, method='fdr_bh')[1]
     r = r[['cog', 'category', 'pvalue', 'qvalue', 'empirical-qvalue']]
-
+    pathway_genes_dict = m.groupby('COG_category')['Preferred_name'].apply(list).to_dict()
```

Comment:
I am a bit confused here because you talk about pathways, but you are looking at COG categories here?